libraries/lib-files/FileNames.h

```diff
@@ -204,7 +204,8 @@ namespace FileNames
 
 
    //! Check location on writable access and return true if checked successfully.
-   FILES_API bool WritableLocationCheck(const FilePath& path);
+   FILES_API bool WritableLocationCheck(const FilePath& path,
+                                       const wxString & message);
```

Comment:
message is user-visible.  It should have type TranslatableString.  And you will notice, this change of type will intentionally break compilation of the rest, forcing you to put in the missing `XO()` macro.  That macro is detected by the `msgfmt` program which scans the source code and generates `locale/audacity.pot`.